tools/coqdoc/cpretty.mll

```diff
@@ -828,16 +865,15 @@ and doc indents = parse
           if !Cdglobals.parse_comments then comment lexbuf
           else skipped_comment lexbuf in
         if eol then bol_parse lexbuf else doc indents lexbuf }
-  | '*'* "*)" space_nl* "(**"
-      {(match indents with
+  | '*'* "*)" (space_nl* as s) "(**"
+      { let nl_count = count_newlines s in
+        new_lines nl_count lexbuf;
+       (match indents with
         | Some _ -> Output.stop_item ()
         | None -> ());
        (* this says - if there is a blank line between the two comments,
           insert one in the output too *)
-       let lines = List.length (Str.split_delim (Str.regexp "['\n']") 
-                                                (lexeme lexbuf))
-       in 
-         if lines > 2 then Output.paragraph ();
+         if nl_count > 1 then Output.paragraph ();
```

Comment:
> The links coqdoc test file changed though. Don't know exactly why (a different use of newlines?).

I had an off-by-one mistake here (had left "2" instead of "1").

Reply:
Ah, ok.